tests/UI/tools/stepIdentifiersChecker.ts

```diff
@@ -0,0 +1,190 @@
+import fs from 'fs';
+
+type JsonReportTest = {
+  context?: string,
+  file: string,
+  pending: boolean,
+  fullTitle: string
+}
+type JsonReportSuite = {
+  file: string,
+  tests: Array<JsonReportTest>
```

Comment:
```suggestion
  tests: Array<JsonReportTest>,
```

or remove every `,` to stay consistent, as you prefer

Reply:
Fixed